### ISSUE TITLE
SPARQL Import as a separate extension (like it was at RoamJS.com)

### ISSUE THREAD

**Issue by kvistgaard** [dvargas92495/roamjs-developer#17]
Please consider having SPARQL import as a separate extension like it was before.

Here are a few arguments:
1. It's not for developers. I'm not a developer myself, and many of the people I know who use SPARQL are not either. And anyway, in the default mode, the extension does not require knowing SPARQL or any technical skill
2. It will increase the extension visibility and discoverability (now in the Depot if you search "SPARQL", nothing will be retrieved)
3. It won't require javascript eval, so potentially, more people will try it out and start using it
4. To make it more useful, the user should be able to select the desired WD properties, but such a development will only be justified if there are more users, so separating it might start such virtuous cycles.

**Comment by mdroidian** [comment]
Absolutely, I understand where you're coming from with your suggestion to revert the SPARQL import to a standalone extension. Your points are valid, particularly regarding the ease of use for non-developers and the potential for increased visibility and user adoption. 

Here's a bit of insight into why things are the way they are currently:

- **Streamlining Development**: One of the main reasons the extensions were consolidated  was to streamline the development process. Maintaining a plethora of repositories can be quite taxing, especially with limited resources.

- **Community Contribution**: On the bright side, the open-source nature of our work means that community members are always welcome to contribute. If there's a general consensus that a separate SPARQL extension would be beneficial, anyone in the community can fork the relevant code and create it. We're all for community-driven initiatives that can enhance the ecosystem.

- **Collaborative Opportunities**: If this is a project close to your heart, and you're interested in spearheading the initiative to separate out the SPARQL functionality, I'm open to discussing freelance opportunities. ``
